Loader/SymfonyFixturesLoader.php

```diff
@@ -72,6 +79,48 @@ protected function createFixture($class)
         return $this->loadedFixtures[$class];
     }
 
+    /**
+     * Returns the array of data fixtures to execute.
+     *
+     * @param string $set
+     *
+     * @return array $fixtures
+     */
+    public function getFixtures($set = null)
+    {
+        $fixtures = parent::getFixtures();
+
+        if ($set) {
+            $filteredFixtures = [];
+            foreach ($fixtures as $class => $fixture) {
```

Comment:
I see you changed from `array_map` to a `foreach` loop. You could also use `array_walk`. But -`foreach` is fine if you prefer that.

Reply:
since i am creating a new array anyway i choose foreach so i dont need to use `use`.
don't know which one is the fastest array_walk or foreach

Reply:
Well, you do not need to create a new array, it is ok to change the local variable `$fixtures `.

But it is up to you, this PR is fine either way.